src/hnasko_lab_to_nwb/embargo_2025/interfaces/demodulated_tdt_interface.py

```diff
@@ -103,12 +104,10 @@ def add_to_nwbfile(
         except KeyError:
             raise ValueError("Fiber photometry metadata not found in NWB file.")
 
-
         # Create a new processing module for fiber photometry signals if it doesn't exist
         if "ophys" not in nwbfile.processing:
             ophys_module = nwbfile.create_processing_module(
-                name="ophys",
-                description="Processed fiber photometry signals"
+                name="ophys", description="Processed fiber photometry signals"
```

Comment:
you can replace lines 108-113 with the `get_module()` function from neuroconv. 
then this will be : `get_module("ophys", description="Processed fiber photometry signals")`